.github/workflows/release-docs.yml

```diff
@@ -23,11 +23,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: doc
-
-      - uses: actions-rs/cargo@v1
-        with:
-          command: doc
-          args: --bins
+          args: --bins --lib
```

Comment:
Is there a way to not build docs for all the crates and only for `stabilizer, miniconf, ad9959, dsp`?

Reply:
And running something like [cargo-deadlinks](https://github.com/deadlinks/cargo-deadlinks) would be useful as well.